horace_core/sqw/coord_transform/@CurveProjBase/CurveProjBase.m

```diff
@@ -0,0 +1,245 @@
+classdef CurveProjBase <aProjectionBase
+    % Class defines common properties and method used by cuvelinear
+    % projections. Currently (01/04/2024) spherical and cylindrical
+    % to make spherical cuts.
+    %
+    properties(Dependent)
+        ez;  % [1x3] unit vector specifying crystallographic direction of
+        % spherical coordinates Z-axis within the reciprocal lattice.
+        % Z-axis of spherical coordinate system is the axis where the
+        % elevation angle (MATLAB convention) is counted from.
+        % In Horace/Mantid convention this angle is named theta = pi/2-elevation.
```

Comment:
worth labelling this as polar angle (as opposed to the elevation angle you mention)